test/mocks/components/remaining-amount.js

```diff
@@ -1 +1,11 @@
-module.exports = 200.01
+const topUpRemainingAmount = 200.01
+const reductionRemainingAmount = 200.01
+const reductionZeroRemainingAmount = 0

```

Comment:
`0.00` for explicit 'we expect this to be a float' (I think it'll get cast to an int anyway)

Reply:
integer 0 is a member of numeric type, hence 0 value is valid, for better readability, I have changed to 0.00